src/abc_inference.jl

```diff
@@ -0,0 +1,24 @@
+function createabcfunction(prob, t, distancefunction, alg, kwargs...)
+    function simfunc(params, constants, targetdata)
+        sol = solve(problem_new_parameters(prob, params), alg, kwargs...)
+        randomized = VectorOfArray([sol(t[i]) for i in 1:length(t)])
```

Comment:
with `saveat` set, you can avoid the interpolation and just use `convert(Array,sol)` directly.